app/helpers/moments_form_helper.rb

```diff
@@ -87,7 +87,8 @@ def moment_bookmarked
       'bookmarked', 'switch', 'moments.form.bookmarked_question'
     ).merge(
       value: true, uncheckedValue: false,
-      checked: @moment.bookmarked, dark: true
+      checked: @moment.bookmarked, dark: true,
```

Comment:
`checked: @moment.bookmarked` should be `checked: params[:bookmarked] ? true : @moment.bookmarked` so that when you create a Moment from the Care Plan page, it will automatically be bookmarked.

Reply:
Yeah gotcha, updated now.